src/nimqml/private/qmodelindex.nim

```diff
@@ -8,7 +8,7 @@ proc setup(self: QModelIndex, other: DosQModelIndex, takeOwnership: Ownership) =
 
 proc delete*(self: QModelIndex) =
   ## Delete the given QModelIndex
-  if not self.vptr.isNil:
+  if self.vptr.isNil:
```

Comment:
This looks like a massive leak, given a lot of model indexes are instantiated in our models.

Reply:
Nobody has ever check the repo we forked from.
It's still there: https://github.com/filcuc/nimqml/blob/master/src/nimqml/private/qmodelindex.nim#L9-L15

@alexjba you can open a pull request there as well.

Reply:
@saledjenic good point. I will